README.md

```diff
@@ -2,79 +2,126 @@
 
 > A starter project for reading beacon values from a smart contract
 
-This project is composed of the following:
-
-1. A test that uses a mocked RrpBeaconServer to simulate reading a beacon value from a smart contract.
-1. A script to deploy the BeaconReaderExample contract to any network. This script will set the address of the RrpBeaconServer contract deployed on the selected network and after the script finishes it will display the address of the contract deployed but also save this address to `deployments` directory on the root of this repo (except for hardhat network since this network is started and stop just for the duration of the script execution).
-1. A script to read the beacon value from the deployed contract. If selected network is not `hardhat` or `localhost` then this address must be then whitelisted by the RrpBeaconServer manager account prior to reading the beacon value.
+This project was created by the `create-beacon-reader-app` CLI from the
+[services repo](https://github.com/api3dao/services). This project is composed
+of the following:
+
+1. A test that uses a mocked `RrpBeaconServer` contract to simulate reading a
```

Comment:
I would add:
```
on `localhost`
```
to after "contract"

Reply:
You can actually run those test on non localhost network as well (if you specify `--network ropsten` after `yarn test`).